backend/src/main/scala/com/softwaremill/bootzooka/http/HttpApi.scala

```diff
@@ -59,6 +61,12 @@ class HttpApi(
 
   private lazy val corsConfig: CORSConfig = CORS.DefaultCORSConfig
 
+  private def forwardToRootResponse(request: Request[Task]): Response[Task] = {
+    Response[Task]()
+      .withEntity(Source.fromResource("webapp/index.html").getLines().mkString)
+      .withHeaders(request.headers)
```

Comment:
one thing I'm wondering about: why do you need to mirror the request headers in the response?

Reply:
Apparently, this is not necessary. The code below also works. I think, correctly need to do so:
```
private def forwardToRootResponse(request: Request[Task]): Response[Task] = {
    Response[Task]()
      .withEntity(Source.fromResource("webapp/index.html").getLines().mkString)
      .withHeaders(Location(Uri.uri("/")))
  }
```

Reply:
Good to know, I'll change it :) But do you know why is this required? I'd like to add a comment with an explanation

Reply:
In original commit before force commit I tried create `302 Found` response with `/` location but without content because client should ignore `302 Found` body. It is not works, it just redirect to `/` instead of requested URI. When I added `.withEntity(Source.fromResource("webapp/index.html").getLines().mkString)` it worked.  So change it to:
```
private def forwardToRootResponse(request: Request[Task]): Response[Task] = {
    Response[Task]()
      .withStatus(Status.Found)
      .withEntity(Source.fromResource("webapp/index.html").getLines().mkString)
      .withHeaders(Location(Uri.uri("/")))
  }
```
You can read here about `302 Found` and `Location` header: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/302

Reply:
Shortly (and can be used as comment):
As I understand it, this should work (simple `302 Found` redirect to `Location`):
```
    Response[Task]()
      .withStatus(Status.Found)
      .withHeaders(Location(Uri.uri("/")))
```
But it doesn't work. Maybe it's an http4s bug. Fixed adding body with `.withEntity(Source.fromResource("webapp/index.html").getLines().mkString)`

Reply:
From [Wikipedia](https://en.wikipedia.org/wiki/HTTP_location):
> The HTTP Location header field is returned in responses from an HTTP server under two circumstances: 
> 1. ...
> 2. To provide information about the location of a newly created resource. In this circumstance, the Location header should be sent with an HTTP status code of 201 or 202.

You can return `200 OK` with content and `Location`. For this reason the following code will work:
```
private def forwardToRootResponse: Response[Task] = {
    Response[Task]()
      .withEntity(Source.fromResource("webapp/index.html").getLines().mkString)
      .withHeaders(Location(Uri.uri("/")))
  }
```
The merged code also works because the `Location` is copied from the original request.

Reply:
thanks, I added the note to the code :)